packages/netlify-cms-widget-list/src/ListControl.js

```diff
@@ -104,26 +113,6 @@ export default class ListControl extends React.Component {
     return true;
   }
 
-  componentDidMount() {
-    const { field } = this.props;
-
-    if (field.get('fields')) {
-      this.valueType = valueTypes.MULTIPLE;
-    } else if (field.get('field')) {
-      this.valueType = valueTypes.SINGLE;
-    }
```

Comment:
Is this really slow enough to need memoization, or can we just check from the props every time (like I've changed it to here)?

Reply:
Running it every time should be fine, agreed.